File: abc164/b/b.c
#include <stdio.h>

char *battle(int a, int b, int c, int d) {
  while (1) {
    c -= b;
    if (c <= 0) {
      return "Yes";
    }
    a -= d;
    if (a <= 0) {
      return "No";
    }
  }
}

int main(void) {
  int a, b, c, d;
  scanf("%d %d %d %d", &a, &b, &c, &d);

  printf("%s\n", battle(a, b, c, d));

  return 0;
}

File: abc164/b/b.php
<?php 
function battle(int $a, int $b, int $c, int $d): string {
  while(true){
    $c -= $b;
    if($c <= 0){
      return "Yes";
    }
    $a -= $d;
    if($a<= 0){
      return "No";
    }
  }
}

[$a, $b, $c, $d] = array_map('intval', explode(" ", trim(fgets(STDIN))));

echo battle($a, $b, $c, $d) . "\n";

